docs/advanced/experimental/indexers-cheat-sheet.md

```diff
@@ -0,0 +1,43 @@
+# Indexers on Jina Hub
+
+The [Jina Hub](hub.jina.ai) offers multiple Indexers for different use-cases.
+There are four major questions that should be answered, when deciding for an Indexer and its configuration.
+In a lot of production use-cases Indexers heavily use {ref}`shards <./scaling/shards>` and {ref}`replicas <./scaling/replicas>`
```

Comment:
I don't think this is the correct syntax. The part between `<>` needs to be the id of another section. Like, e.g. https://github.com/jina-ai/jina/blob/660cb3e8b9deba9826b3f7a5d29b2c99b4cf6776/docs/fundamentals/document/documentarraymemmap-api.md#L73 and then you use it like https://github.com/jina-ai/jina/blob/660cb3e8b9deba9826b3f7a5d29b2c99b4cf6776/docs/fundamentals/document/documentarraymemmap-api.md#L69

Reply:
I thought cross-file you need to link the file as well. I'll change it.